### PR TITLE
feat(list): support onRow

### LINKED ISSUE #2213: 🧐ProList 怎么设置行的点击事件？？？

### PR DESCRIPTION
**Pull request by DerrickTel**
close #2213

### CODE REVIEW

File: packages/list/src/ListView.tsx [hunk 1]
```diff
@@ -3,6 +3,7 @@ import type { ListProps, TableColumnType, TableProps } from 'antd';
 import { List } from 'antd';
 import type { GetRowKey } from 'antd/lib/table/interface';
 import type { ActionType } from '@ant-design/pro-table';
+import type { GetComponentProps } from '@ant-design/pro-list';
```
Comment (DerrickTel) [inline packages/list/src/ListView.tsx:6]: 要改成`import type { GetComponentProps } from './index';`吗？

Comment (chenshuai2144) [reply packages/list/src/ListView.tsx:6]: 改一下把，组件内部就不要套娃了

**Comment by chenshuai2144** [comment]
我还想的加个 onClick ，看起来这个是最好的

**Comment by fleebt** [comment]
ant-design/pro-list where use grid property to show as card list, the onrow do not work  any more

**Pull request merged**